### PR TITLE
Displays message when missing a template engine for benchmarking.

### PR DESCRIPTION
**Pull request by jnlopar**
Currently it just avoids missing engines silently. This was confusing when running it and wondering where Spitfire itself was. :P This should point users in the right direction.

### CODE REVIEW

File: tests/benchmarks/render_benchmark.py [hunk 1]
```diff
@@ -471,6 +483,13 @@ def main():
                              default=False)
     (options, args) = option_parser.parse_args()
 
+    missing_engines = _get_missing_template_engines()
```
Comment (nicksay) [inline tests/benchmarks/render_benchmark.py]: This means that this message will always show even if not attempting to compare, etc, right?  Alternatively you could change `run_tests` to track which `get_ENGINE_tests` calls return empty lists and print the warning before the tests are run.

**Comment by jnlopar** [comment]
Ok, I went with the `run_tests` approach.

**Comment by awbraunstein** [comment]
lgtm

**Comment by nicksay** [comment]
LGTM

**Pull request merged**